datadog_test.go

```diff
@@ -0,0 +1,69 @@
+package gzap
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest/observer"
+
```

Comment:
Gofmt doesn't like the space here

Reply:
Shouldn't this be built into the build process?

Reply:
It should and I thought it did but I guess it doesn't. The linting initiative didn't get to it with the linting standards yet, but hope to get to the go projects soon.